InAppMessaging/Example/Tests/FIRIAMDisplayExecutorTests.m

```diff
@@ -28,44 +28,57 @@ @interface FIRIAMMessageContentDataForTesting : NSObject <FIRIAMMessageContentDa
 @property(nonatomic, readwrite, nonnull) NSString *titleText;
 @property(nonatomic, readwrite, nonnull) NSString *bodyText;
 @property(nonatomic, nullable) NSString *actionButtonText;
+@property(nonatomic, nullable) NSString *secondaryActionButtonText;
 @property(nonatomic, nullable) NSURL *actionURL;
+@property(nonatomic, nullable) NSURL *secondaryActionURL;
 @property(nonatomic, nullable) NSURL *imageURL;
+@property(nonatomic, nullable) NSURL *landscapeImageURL;
 @property BOOL errorEncountered;
 
 - (instancetype)initWithMessageTitle:(NSString *)title
                          messageBody:(NSString *)body
                     actionButtonText:(nullable NSString *)actionButtonText
+           secondaryActionButtonText:(nullable NSString *)secondaryActionButtonText
                            actionURL:(nullable NSURL *)actionURL
+                  secondaryActionURL:(nullable NSURL *)secondaryActionURL
                             imageURL:(nullable NSURL *)imageURL
+                   landscapeImageURL:(nullable NSURL *)landscapeImageURL
                        hasImageError:(BOOL)hasImageError;
 @end
 
 @implementation FIRIAMMessageContentDataForTesting
 - (instancetype)initWithMessageTitle:(NSString *)title
                          messageBody:(NSString *)body
                     actionButtonText:(nullable NSString *)actionButtonText
+           secondaryActionButtonText:(nullable NSString *)secondaryActionButtonText
                            actionURL:(nullable NSURL *)actionURL
+                  secondaryActionURL:(nullable NSURL *)secondaryActionURL
                             imageURL:(nullable NSURL *)imageURL
+                   landscapeImageURL:(nullable NSURL *)landscapeImageURL
                        hasImageError:(BOOL)hasImageError {
   if (self = [super init]) {
     _titleText = title;
     _bodyText = body;
     _imageURL = imageURL;
+    _landscapeImageURL = landscapeImageURL;
     _actionButtonText = actionButtonText;
+    _secondaryActionButtonText = secondaryActionButtonText;
     _actionURL = actionURL;
+    _secondaryActionURL = secondaryActionURL;
     _errorEncountered = hasImageError;
   }
   return self;
 }
 
 - (void)loadImageDataWithBlock:(void (^)(NSData *_Nullable imageData,
+                                         NSData *_Nullable landscapeImageData,
                                          NSError *_Nullable error))block {
   if (self.errorEncountered) {
-    block(nil, [NSError errorWithDomain:@"image error" code:0 userInfo:nil]);
+    block(nil, nil, [NSError errorWithDomain:@"image error" code:0 userInfo:nil]);
   } else {
     NSString *str = @"image data";
     NSData *data = [str dataUsingEncoding:NSUTF8StringEncoding];
-    block(data, nil);
+    block(data, nil, nil);
```

Comment:
Should we be testing various differences with landscapeImageData as well?

Reply:
Oops :)